docs/examples/resnet50_trt/README.md

```diff
@@ -0,0 +1,247 @@
+# Triton Inference Server with DALI backend
+
+This repo is an example of [DALI (Data Loading Library) backend](https://github.com/NVIDIA/DALI) for image classification on [Triton Inference server](https://github.com/triton-inference-server/server).
+
+ResNet50 model optimized by [TensorRT](https://developer.nvidia.com/tensorrt) is used for image classification. 
+
+
+#### Dependencies
+
+* Export ONNX and build TensorRT
+  * nvcr.io/nvidia/pytorch:20.12-py3
+* Triton Inference Server for DALI backend
+  * nvcr.io/nvidia/tritonserver:20.12-py3
+  * https://github.com/triton-inference-server/dali_backend
+
+* Client
+  * nvcr.io/nvidia/tritonserver:20.10-py3-clients
+
+#### Setting up the ONNX-TensorRT and DALI backend ENV
+
+```
+$git clone https://gitlab-master.nvidia.com/ghong/triton_dali
+$cd triton_dali
+$git clone https://github.com/triton-inference-server/dali_backend
+$cd dali_backend
+$docker build -t triton_dali_backend -f Dockerfile .
+$cd ..
+```
+
+####  Build TensorRT via ONNX
+
+##### 1.  Converting PyTorch Model to ONNX-model 
+
+Run `onnx_exporter.py` for conversion using PyTorch model to ONNX model. In this case, ResNet50 model is converted to ONNX format. `width` and `height` dims are fixed at 224 but dynamic axes arguments for dynamic batch is used. 
+
+```
+$docker run -it --gpus=all -v $(pwd):/workspace nvcr.io/nvidia/pytorch:20.12-py3 bash
+$python onnx_exporter.py --save model.onnx
+```
+
+##### 2. Building ONNX-model to TensorRT engine
+
+Set the arguments for enabling fp16 precision `--fp16` and for dynamic shapes using a profile `--minShapes`, `--optShapes`, and `maxShapes` with `--explicitBatch`
+
+```python
+$trtexec --onnx=model.onnx --saveEngine=./model_repository/resnet50_trt/1/model.plan --explicitBatch --minShapes=input:1x3x224x224 --optShapes=input:1x3x224x224 --maxShapes=input:256x3x224x224 --fp16
```

Comment:
This command ends with error because model_repository/resnet50_trt/__1__ directory is lacking.

Reply:
Add commands for creating directories. [README.md](https://github.com/triton-inference-server/dali_backend/pull/35/commits/e64638fc5015b57260cd114e6723c81fdb493744#diff-3272a2f918db29f27589316303368b47f9187f0910d33ec7fb7d6f18868ceea2R42)